docs/development/updating-dependencies.md

```diff
@@ -33,14 +34,30 @@ npm install vue-cli-plugin-tauri@latest
 
 ### Update Cargo Packages
 
-Go to `src-tauri/Cargo.toml` and change `tauri` to
-`tauri = { version = "%version%" }` where `%version%` is the version number shown above. (You can just use the `MAJOR.MINOR`) version, like `0.9`.
+You can check for outdated packages with [`cargo outdated`] or on the crates.io pages: [tauri] / [tauri-build].
```

Comment:
Thank you for fixing up the `cargo outdated` link -- was about to push up a PR to fix that :)